README.md

```diff
@@ -60,9 +60,7 @@ message Node {
 
 ``` protobuf
 message Interface {
-  string name = 1;
-  int32 major = 2;
-  int32 minor = 3;
+ bytes json_value = 1;
```

Comment:
Do we still need the `Interface` message if it just has a single field as content? e.g. in RealmManagement protobuf messages we just use a `bytes interface_json` field.